### PR TITLE
fix: don't return out of missing cookie handler

### PR DESCRIPTION
**Pull request by eadmundo**
review in conjunction with autotelic/fasdentify#241

### CODE REVIEW

File: lib/createCookieEncryptor.js [hunk 1]
```diff
@@ -73,12 +73,15 @@ export const createCookieEncryptor = (opts) => {
         obj[name] = value
         return obj
       }, {})
-      if (!cookieObj.hasOwnProperty(cookieName)) {
-        return missingCookieHandler(request, event)
+      const hasCookie = cookieObj.hasOwnProperty(cookieName)
+      if (!hasCookie) {
+        missingCookieHandler(request, event)
+      }
+      if (hasCookie) {
```
Comment (CLSnazel) [inline lib/createCookieEncryptor.js:80]: Is there a reason this is two separate conditionals? Is there a case where both conditionals should run?

Comment (eadmundo) [reply lib/createCookieEncryptor.js:80]: good q!

Comment (eadmundo) [reply lib/createCookieEncryptor.js]: good catch

**Comment by eadmundo** [comment]
force push to swap the boolean to `hasCookie`

**Review by CLSnazel — APPROVED** [review]
LGTM!

I can't seem how to figure out how to make the fasdentify gateway pull and install these new changes to test it again, but I trust that it's working. 👍

**Comment by eadmundo** [comment]
> I can't seem how to figure out how to make the fasdentify gateway pull and install these new changes to test it again

uninstall and reinstall :/

**Comment by eadmundo** [comment]
> but I trust that it's working.

it is! I tested it 😅

**Pull request merged**